runtime/near-vm-runner/Cargo.toml

```diff
@@ -14,11 +14,11 @@ This crate implements the specification of the interface that Near blockchain ex
 
 [dependencies]
 cached = "0.12"
-wasmer-runtime = { version = "=0.17.1", features = ["default-backend-singlepass"], default-features = false }
-wasmer-runtime-core = { version = "=0.17.1" }
+wasmer-runtime = { version="0.17.1", features = ["default-backend-singlepass"], default-features = false, package = "wasmer-runtime-near" }
+wasmer-runtime-core = {version = "0.17.1", package = "wasmer-runtime-core-near" }
 pwasm-utils = "0.12"
 parity-wasm = "0.41"
-wasmtime = { version = "0.17.0", features = ["lightbeam"], default-features = false, optional = true }
+wasmtime = { version = "0.20.0", default-features = true, optional = true }
```

Comment:
Why did you enable `default-features`? It seems that it pulls more than we need.

Reply:
Oops!